e3nn/kernel.py

```diff
@@ -57,6 +58,8 @@ def lm_normalization(l_out, l_in):
             for mul_in, l_in, p_in in self.Rs_in:
                 l_filters = self.get_l_filters(l_in, p_in, l_out, p_out)
                 num_summed_elements += mul_in * len(l_filters)
+                list_of_l_filters += [l for l in l_filters for i in range
```

Comment:
don't you want to save more information? like a tuple `(l_in, p_in, l_out, p_out, l)`?

Reply:
My motivation for adding this list is to facilitate plotting kernels. I'm happy for the list to store the full information, but I only need l_filter. See [this in progress function](https://github.com/blondegeek/e3nn_tutorial/blob/volume/spherical.py#L207) (that I hope to add as a method to Kernel) that I'm using to visualize the kernel and [this notebook](https://github.com/blondegeek/e3nn_tutorial/blob/volume/plot_filter.ipynb) for an example of how to use it.

Reply:
As you wish, I merge!